### PR TITLE
Migrating to new apis

### LINKED ISSUE #595: Upgrade component's to MUI ^5.0
### Tasklists -
- [x]  Change the imports as the new API of MUI
- [ ]  Using the new sx prop for styling

### PR DESCRIPTION
**Pull request by arunavabasu-03**
Fixes #595

### CODE REVIEW

File: frontend/.gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 !.env
 !src/components/config.js
 build/
+!radis.d.ts
```
Comment (suzil) [inline frontend/.gitignore:4]: We don't need this. We exclude the ignoring of this file in the parent gitignore. In addition, this isn't the path it's on (it's under `src/`).

**Review by suzil — COMMENTED** [review]
To make this easier to review, would you rebase your commits and adjust it to something like the following:
- First commit: "Rename website/ to frontend/" (and only have changes related to that there)
- Second commit: "Migrate to MUI module imports"

I also think we don't need to add headers to the imports. It should be pretty self-explanatory that an import is a MUI module import that we don't need to label it with a comment.

**Review by suzil — CHANGES_REQUESTED** [review]
Adjust `package.json` yarn install script to do `cd frontend/` instead of `cd website/` as right now that doesn't work.

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "tsc && cd website && yarn build",
```
Comment (suzil) [inline package.json:8]: also here should be "frontend"

**Comment by arunavabasu-03** [comment]
@suzil now all things all ok :)
Hitran             |  Geisa
:-------------------------:|:-------------------------:
Screenshot 2022-07-01 at 7 30 39 PM  |  Screenshot 2022-07-01 at 7 30 23 PM

File: frontend/src/components/fields/MoleculeSelector/index.tsx [hunk 3]
```diff
@@ -60,7 +65,10 @@ export const MoleculeSelector: React.FC<MoleculeSelectorProps> = ({
       onInputChange={(_, newInput) =>
         setInput(addSubscriptsToMolecule(newInput.toUpperCase()))
       }
-      renderOption={(molecule) => addSubscriptsToMolecule(molecule)}
+      renderOption={(props, molecule) => {
+        console.log(molecule);
+        return <li {...props}>{addSubscriptsToMolecule(molecule)}</li>;
```
Comment (arunavabasu-03) [inline frontend/src/components/fields/MoleculeSelector/index.tsx:70]: basically, we render this in this way.

File: frontend/src/components/fields/MoleculeSelector/index.tsx [hunk 4]
```diff
@@ -60,7 +65,10 @@ export const MoleculeSelector: React.FC<MoleculeSelectorProps> = ({
...
+      }}
```
Comment (suzil) [inline frontend/src/components/fields/MoleculeSelector/index.tsx:71]: We don't want the `console.log` on prod here.

```suggestion
      renderOption={(props, molecule) => <li {...props}>{addSubscriptsToMolecule(molecule)}</li>}
```

**Comment by arunavabasu-03** [comment]
@suzil , fixed :)

Screenshot 2022-07-05 at 8 09 37 PM

File: .gitignore [hunk 5]
```diff
@@ -129,5 +129,5 @@ cdk.out
 !config.js
 !lib/inline-lambda/index.js
 *.d.ts
-!website/src/radis.d.ts
+# !website/src/radis.d.ts
 # /Users/arunavabasu/Work/Web/GSOC/radis-app/website/radis.d.ts
```
Comment (suzil) [inline .gitignore:133]:
```suggestion
```
Would be nice to just remove entirely instead of leaving it commented-out.

**Review by suzil — APPROVED** [review]

**Pull request merged**